var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -114,12 +114,13 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     variant('minitensor',   default=False, description='Compile with MiniTensor')
     variant('muelu',        default=True, description='Compile with Muelu')
     variant('nox',          default=False, description='Compile with NOX')
+    variant('panzer',       default=True, description='Compile with Panzer')
```

Comment:
I would suggest the default for panzer be set to False, but agree with the variant be added.